.github/pull_request_template.md

```diff
@@ -1,10 +1,8 @@
-**Please delete this line before submitting**, _Pull request TITLE should look like this_: `[Resource] -> [Resource Section In Docs]`
+# Freebiesbug
```

Comment:
Please revert this file to the original one and only change in the PR body 

Thanks.